app/controllers/instruments_dashboard_controller.rb

```diff
@@ -0,0 +1,35 @@
+class InstrumentsDashboardController < ApplicationController
+
+  layout "plain", only: :public_dashboard
+
+  admin_tab :all
+  before_action :authenticate_user!, except: :public_dashboard
+  before_action :authenticate_token, only: :public_dashboard
+  before_action :init_reservations
+  # This sets "Reservations" as the active tab
+  before_action { @active_tab = "admin_reservations" }
+
+  def dashboard
```

Comment:
I originally went with `index` here but it was causing problems with the active tab logic (and I didn't really feel like refactoring that was worth it in this story).